polkadot/node/core/prospective-parachains/src/fragment_chain/mod.rs

```diff
@@ -1192,7 +1190,7 @@ impl FragmentChain {
 		let Some(mut earliest_rp) = self.earliest_relay_parent() else { return };
 
 		loop {
-			if self.best_chain.chain.len() > self.scope.max_depth {
+			if self.best_chain.chain.len() >= self.scope.max_backable_len {
```

Comment:
Why `>=` here?

Reply:
because we were previously building a chain of maximum `max_candidate_depth + 1`. We're dropping the +1 on the equivalent value now